io-engine/src/bdev/nexus/nexus_io.rs

```diff
@@ -189,6 +190,12 @@ impl<'n> NexusBio<'n> {
                 status,
                 self.ctx()
             );
+            // seems like race condition, ignore the error if the controller is
+            // resetting
+            if NvmeController::reset_active(&child.device_name()) {
+                self.ok();
```

Comment:
is this right? We're oking the IO but it failed?

Reply:
This code was added before I implement the nexus pause/resume. This is not needed now, will remove it